benchmarking/chain-sync/launch_mainnet_proxy.sh

```diff
@@ -11,7 +11,7 @@ set -euo pipefail
 
 cd ${TARGETDIR}
 
-cabal new-run exe:cardano-byron-proxy -- \
+cabal v2-run exe:cardano-byron-proxy -- \
```

Comment:
Shouldn't this change have been part of #384? How many more changes like this did #384 miss?